### ISSUE TITLE
Capture initialization failed

### ISSUE THREAD

**Issue by CataniLorenzo** [pupil-labs/pupil#673]
Hi,
we starting to use Pupil glasses but we found an issue: Capture initialization failed.
Specifically, the first time that we used the device, the cameras works well bu we noticed that the temperature of the three cameras is really high. On a second time, we open Pupil Capture and we saw the image of the world camera and of the right eye camera, but not of the left one.

The screenshot below shows the first messages and image from the three cameras.
![schermata del 2017-03-21 14-42-21](https://cloud.githubusercontent.com/assets/26574324/24150719/d8c98130-0e46-11e7-8f9b-36e9edc19343.png)

Please, help us.
LC

**Comment by papr** [comment]
Dear @CataniLorenzo 
the heat generation is normal. To solve your other problem:

1. Close Pupil Capture
2. Disconnect the headset for at least 5 seconds
3. Reconnect the headset and wait for a few seconds.
4. Start Pupil Capture

Please post our OS and hardware details if the issue still persists after multiple trials of the above procedure. Please test if it is always `EYE1` that fails or if `EYE0` also fails.

**Comment by CataniLorenzo** [comment]
After three trials, the issue persists.
We use Ubuntu 16.10 and Pupil v0.92. The hardware is a Pupil Headset with high speed world camera, and binocular 120hz eye camera.
As mentioned above, in the first use the device worked well. The problem occured on the second use.
From the main window (world camera) we selected first "detect `EYE0`" and the image of the right eye appears but when we activated the `EYE1` the image misses. The problem also occures when we activated first the `EYE1` and then the `EYE0`: only the image on the right camera is showed on the screen.
